### PR TITLE
openmortal: new recipe

### PR DESCRIPTION
**Pull request by Crestwave**
Notes:

- I didn't know anything good I could set the host to so I left it blank.
- I patched the Perl scripts to use relative paths for `require`; is there a better way to handle this?
- It fails to build in strict mode on x86 secondary architecture because it complains that it `can't find used library "libperl.so"`, but it seems to work fine otherwise. I'm guessing this is because it wants a secondary arch library but Perl doesn't have a secondary arch build enabled for some reason; any ideas on what do to here?
- I didn't split the package data as I felt that the difference isn't too much nowadays, but let me know if you feel otherwise.
- How much space should debug info take to warrant a debuginfo package? The debuginfo package takes 1.2 MB on my system, which I'm guessing is enough.

### CODE REVIEW

File: games-arcade/openmortal/openmortal-0.7.recipe [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	devel:libSDL_gfx$secondaryArchSuffix
+	devel:libSDL_image$secondaryArchSuffix
+	devel:libSDL_mixer$secondaryArchSuffix
+	devel:libSDL_net$secondaryArchSuffix
+	devel:libSDL_ttf$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:perl
```
Comment (korli) [inline games-arcade/openmortal/openmortal-0.7.recipe]: please add cmd:sed

Comment (Crestwave) [reply games-arcade/openmortal/openmortal-0.7.recipe]: done

**Comment by korli** [comment]
about libperl.so is there a rpath on the binary?

**Comment by Crestwave** [comment]
Yeah

File: games-arcade/openmortal/openmortal-0.7.recipe [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+{
+	sed -i 's/CChallengeMenu::CChallengeMenu/CChallengeMenu/g' src/OnlineChat.cpp
+	sed -i "s|require '\([A-Z].*\)'|require './\1'|g" data/script/*
+	sed -i "s|require 'DataHelper.pl'|require '../script/DataHelper.pl'|g" data/characters/*.pl
+}
+
+BUILD()
+{
+	runConfigure --omit-dirs "docDir dataRootDir" \
+		./configure --build=x86 --host=
```
Comment (korli) [inline games-arcade/openmortal/openmortal-0.7.recipe]: is --build=x86 valid for x86_64?

Comment (Crestwave) [reply games-arcade/openmortal/openmortal-0.7.recipe]: Yep. --build=x86_64 is also valid for x86 but I felt that was more appropriate. Doesn't seem to do much, as with the host, which works fine with it empty.

**Pull request merged**